bindata/network/ovn-kubernetes/common/ipsec.yaml

```diff
@@ -236,27 +236,28 @@ spec:
             cpu: 10m
             memory: 100Mi
         terminationMessagePolicy: FallbackToLogsOnError
-        readinessProbe:
+        livenessProbe:
           exec:
             command:
             - /bin/bash
             - -c
             - |
               #!/bin/bash
               ovs-appctl -t ovs-monitor-ipsec ipsec/status && ipsec status
-          initialDelaySeconds: 120
-          periodSeconds: 30
-          timeoutSeconds: 40
-        livenessProbe:
+          periodSeconds: 60
+          timeoutSeconds: 20
+        startupProbe:
           exec:
             command:
             - /bin/bash
             - -c
             - |
               #!/bin/bash
               ovs-appctl -t ovs-monitor-ipsec ipsec/status && ipsec status
-          initialDelaySeconds: 60
-          periodSeconds: 60
+          initialDelaySeconds: 15
+          periodSeconds: 30
+          timeoutSeconds: 20
+          failureThreshold: 10
```

Comment:
After 30 seconds * 10 failures, kill and restart the pod  (that gives us 5 minutes)